rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -169,6 +169,23 @@ namespace rsx
 			method_registers.registers[reg] = method_registers.register_previous_value;
 		}
 
+		void set_notify(thread* rsx, u32 _reg, u32 arg)
+		{
+			const u32 location = method_registers.context_dma_notify();
+			const u32 index = (location & 0x7) ^ 0x7;
+
+			if ((location & ~7) != (CELL_GCM_CONTEXT_DMA_NOTIFY_MAIN_0 & ~7))
+			{
+				// TODO: Gcm sets the default to CELL_GCM_CONTEXT_DMA_TO_MEMORY_GET_NOTIFY0
+				fmt::throw_exception("NV4097_NOTIFY: Unimplemented/invalid context = 0x%x" HERE, method_registers.context_dma_notify());
+			}
+
+			rsx->sync();
```

Comment:
Do not execute sync lightly, it can stall rsx for several milliseconds at a time. It is not necessary here since we do not need a ROP sync / cache invalidate.